core/storage/opportunity/gae_models.py

```diff
@@ -106,6 +107,8 @@ def get_all_translation_opportunities(
                 of the full list of entities.
             language_code: str. The language for which translation opportunities
                 are to be fetched.
+            topic_name: str. The topic for which translation opportunities
+                should be fetched.
```

Comment:
Can you specify that this can be None as well, and the expected behavior if it is?